src/Toolkit/Toolkit.WinUI/UI/Controls/MeasureToolbar/MeasureToolbar.Theme.xaml

```diff
@@ -4,14 +4,26 @@
     xmlns:controls="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
     <ResourceDictionary.MergedDictionaries>
         <ResourceDictionary Source="MeasureToolbar.Resources.xaml" />
+        <ResourceDictionary x:Key="Default">
+            <SolidColorBrush x:Key="BarBackground"  Color="#eaeaeaea" />
+            <SolidColorBrush x:Key="BarBorder" Color="#DFDFDF" />
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="Light">
+            <SolidColorBrush x:Key="BarBackground"  Color="#eaeaeaea" />
+            <SolidColorBrush x:Key="BarBorder" Color="#DFDFDF" />
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="Dark">
+            <SolidColorBrush x:Key="BarBackground" Color="#ea151515" />
+            <SolidColorBrush x:Key="BarBorder" Color="#404040" />
+        </ResourceDictionary>
     </ResourceDictionary.MergedDictionaries>
     <Style TargetType="controls:MeasureToolbar">
         <Setter Property="Background"
-                Value="{ThemeResource ComboBoxBackground}" />
+                Value="{ThemeResource BarBackground}" />
         <Setter Property="Foreground"
                 Value="{ThemeResource ComboBoxForeground}" />
         <Setter Property="BorderBrush"
-                Value="{ThemeResource ComboBoxBorderBrush}" />
+                Value="{ThemeResource BarBorder}" />
```

Comment:
Do the other platforms have a matching set of resource names?

Reply:
WPF sets the value directly since it doesn't need to account for light/dark mode.